File: 1550_Three_Consecutive_Odds.js
var threeConsecutiveOdds = function(arr) {
    let i = 0;
    while(i<arr.length){
        let count = 0;
        if(arr[i] % 2 !== 0){
            let j = i+1;    
            while(count<3 && j<arr.length){
                if(arr[j] % 2 === 0) break;
                else count++;
                j++;
            }
            if(count === 2) return true;
        }
        i++;
    }
    return false;
};

File: 1550_Three_Consecutive_Odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int x = 0, i = 0;
        while(i < arr.size()){
            if(arr[i] % 2 != 0) x++;
            else x = 0;
            if(x == 3) return true;
            i++;
        }
        return false;
    }
};
